### DISCUSSION TITLE
What is the word for ... (so that I can search for it)?

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14171] [Q&A] [answered]
The contractor wrote
```xaml
        <SolidColorBrush Color="Gray" x:Key="NavigationViewItemBackground"/>
        <SolidColorBrush Color="DarkGray" x:Key="TopNavigationViewItemBackgroundPointerOver"/>
        <SolidColorBrush Color="DarkSlateGray" x:Key="TopNavigationViewItemBackgroundSelected"/>
        <SolidColorBrush Color="DarkSlateGray" x:Key="TopNavigationViewItemBackgroundSelectedPointerOver"/>
        <SolidColorBrush Color="DarkGray" x:Key="TopNavigationViewItemBackgroundPressed"/>
        <SolidColorBrush Color="DarkSlateGray" x:Key="TopNavigationViewItemBackgroundSelectedPressed"/>
```
What do you call the thing in the `x:Key`? I want to be able to search for the one that controls the background colour of RichTextBoxes but asking for `the key of solidcolorbrush` isn't specific enough for Google it seems. If it had a name, like 'notUniquelyNamedAttribute', then searching for `the notUniquelyNamedAttribute of solidcolorbrush` might give me better results.

**Comment by axtens** [comment, marked as answer]
Found what I needed in my local `.nuget\packages\microsoft.windowsappsdk\1.4.231008000\lib\net6.0-windows10.0.18362.0\Microsoft.WinUI\Themes\generic.xaml`

**Reply by jeromelaban** [reply]
What you're searching for are called resource keys, and are generally found [here](https://github.com/unoplatform/uno/tree/master/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault) in uno, and here in the official [WinUI sources](https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.4-stable).
